runtime/src/lib.rs

```diff
@@ -129,10 +127,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // and set impl_version to 0. If only runtime
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
-    spec_version: 240,
+    spec_version: 1000,
     impl_version: 0,
     apis: RUNTIME_API_VERSIONS,
-    transaction_version: 1,
+    transaction_version: 2,
```

Comment:
Since our spec name is the same as the standalone, it seemed worthwhile to bump it to indicate that the parachain version will have slightly different pallets than the standalone.